lsp/nls/src/requests/goto.rs

```diff
@@ -123,21 +123,25 @@ pub fn handle_to_usages(
 
             for reference_id in usages.iter() {
                 let reference = linearization.get_item(*reference_id).unwrap();
-                let location = match reference.pos {
-                    RawSpan {
-                        start: ByteIndex(start),
-                        end: ByteIndex(end),
-                        src_id,
-                    } => Location {
-                        uri: Url::parse(&server.cache.name(src_id).to_string_lossy()).unwrap(),
-                        range: Range::from_codespan(
-                            &src_id,
-                            &(start as usize..end as usize),
-                            server.cache.files(),
-                        ),
-                    },
-                };
-                locations.push(location);
+                if reference.pos == TermPos::None {
```

Comment:
Nitpick: I would use pattern matching or `into_opt` (even if the latter may cause an allocation, if not optimized away).